src/client/render/core.cpp

```diff
@@ -32,17 +32,24 @@ RenderingCore::RenderingCore(IrrlichtDevice *_device, Client *_client, Hud *_hud
 {
 	screensize = driver->getScreenSize();
 	virtual_size = screensize;
+
+	shadow_renderer = new ShadowRenderer(device, client);
```

Comment:
instead of initializing there, initialize the variable just upper like client or hud

Reply:
Done.